app/utils.ts

```diff
@@ -255,6 +255,8 @@ export function isVisionModel(model: string) {
     "claude-3",
     "gemini-1.5-pro",
     "gemini-1.5-flash",
```

Comment:
"gemini-1.5-pro",
 "gemini-1.5-flash"
 
Is it the same as above?

Reply:
"gemini-1.5-pro" and "gemini-pro-1.5" are the same model, just named differently by different vendors.

Reply:
see https://openrouter.ai/models/google/gemini-pro-1.5